backport/createStatusComment.js

```diff
@@ -3,7 +3,8 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.getCommentFromResponse = void 0;
```

Comment:
Having transpiled files in source control feels odd. I was prompted to fix conflicts twice (in both js and ts files) - obviously only did it in ts files.